reactnative/reactnative-todo/ios/reactnativetodo/FRAuthSampleStructs.swift

```diff
@@ -110,6 +113,13 @@ public struct FRCallback: Encodable {
           print(error)
         }
       }
+      // Serialize the server response into a dictionary for react native layer
+      if let thisCallback = callback as? DeviceProfileCallback {
```

Comment:
Do we need to pass the deviceProfileCallback result to the UI?
Usually the deviceProfileCallback when encountered is executed automatically from the app submitted. I don't have anything against this approach, just wondering if there is value for the RN side of the app to get this